### PR TITLE
Added the IssuedAt property

### PR DESCRIPTION
**Pull request by macaba**

### CODE REVIEW

**Comment by brentschmaltz** [comment]
@macaba thanks. We need you to sign a 'Contributor License Agreement' above.

File: src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs [hunk 1]
```diff
@@ -299,6 +299,18 @@ public DateTime ValidTo
             }
         }
 
+		/// <summary>
```
Comment (GeoK) [inline src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs]: Hi @macaba, thanks for the contribution.
The formatting is slightly off here and at the same spot in JwtSecurityToken class.

Comment (macaba) [reply src/System.IdentityModel.Tokens.Jwt/JwtPayload.cs]: Thanks, resolved.

**Comment by brentschmaltz** [comment]
@macaba thanks for your contribution :-)

**Pull request merged**